config/routes.rb

```diff
@@ -7,6 +7,24 @@
 
   get 'users/styleguide' => 'users#styleguide', as: :signup
 
-  resources :organization_campaigns, only: :show
-  resources :matches, only: :create
+  resources :memberships, only: [:index, :new, :create, :edit, :update, :delete]
+  resources :matches, only: [:new, :create, :delete]
+  resources :organizations, only: [:show] do
+    member do
+      get :admins
+      get :new_admin
+      get :switch_current_campaign
+      get :import_emails_form
+      post :import_emails_form_page2
+    end
+  end
+  resources :organization_campaigns, only: [:show,:create] do
+    member do
+      get :send_email_form
+      get :unmatched
+      get :unmatched_tags
+      get :matched
+      get :match_form
+    end
+  end
 end
```

Comment:
Yeah, I can see where `get :unmatched` above would be better served by a `GET /recipients?organization_campaign_id=1&unmatched`.  I'll do some restructuring along those lines.  But human-fillable forms like `send_email_form` really don't have a REST equiv do they?  (They're like :new and :edit in the standard Rails controller verbs).